resources/python/make_docker.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+#
+# Copyright (C) 2019 Ultimaker B.V.
+#
+
+set -eu
+
+GIT_HASH="$(git rev-parse HEAD)"
+
+DOCKER_IMAGE_NAME="${1:-opinicus_${GIT_HASH}}"
```

Comment:
Seems like an image name specific to a certain repository. Should this be added to the Opinicus repository instead?